### DISCUSSION TITLE
Welcome to Boxedwine Discussions!

### DISCUSSION THREAD

**Discussion by danoon2** [danoon2/Boxedwine#7] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jsigle** [comment]
Hi danoon2
Thanks a lot for the very nice boxedwine work. I've just tried it to experiment with bringing a program originally made for MS Windows / Object Pascal into a web browser.

Now I wanted the Wine desktop and program window to appear in 1024x768 pixels instead of the default 800x600. For the linux version, the -resolution or -fullscreen parameters work, but they seem to be missing for the emscripten version. 

I managed to set the canvas size via css / html / or Javascript patches - but the Wine running inside just doesn't care (it uses only 800x600 of the 1024x768 canvas). Adding several entries to Wine's user.reg in the wine...min.zip didn't help either. And changing e.g.  knativewindow.cpp and then rebuilding Boxedwine/project/emscripten only gives me a version that fails with an error message during startup. I attempted that from various branches, so I'm somewhat stuck here.

So before I go on trying - gould you give me any hint whether this can be achieved in a simple way that I have overlooked so far?
I can also provide more details (like system environment, tried branches, error messages etc.) from my attempts if you like.

Thank you very much for any response and kind regards! Joerg

**Reply by kevodwyer** [reply]
I can look at adding resolution parmater to emscripten build this weekend.

**Reply by kevodwyer** [reply]
ok - resolution parameter is now available from emscripten use &resolution=1024x768 in url to configure
